src/System.Linq.Expressions/src/Properties/InternalsVisibleTo.cs

```diff
@@ -0,0 +1,9 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+//------------------------------------------------------------
+
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("System.Linq.Expressions.Tests, PublicKey=002400000480000094000000060200000024000052534131000400000100010015c01ae1f50e8cc09ba9eac9147cf8fd9fce2cfe9f8dce4f7301c4132ca9fb50ce8cbf1df4dc18dd4d210e4345c744ecb3365ed327efdbc52603faa5e21daa11234c8c4a73e51f03bf192544581ebe107adee3a34928e39d04e524a9ce729d5090bfd7dad9d10c722c0def9ccc08ff0a03790e48bcd1f9b6c476063e1966a1c4")]
```

Comment:
There are 5 projects that using `InternalsVisibleTo` to expose "internals" to Tests project:
1. System.Xml.XDocument
2. System.Reflection.Metadata
3. System.Diagnostics.Debug
4. System.Collections.Immutable
5. System.Text.Encodings.Web

That's why I applied that attribute to `System.Linq.Expressions`. Do I need to remove it?